jpa/src/main/java/org/jboss/as/jpa/container/TransactionScopedEntityManager.java

```diff
@@ -22,6 +22,8 @@
 
 package org.jboss.as.jpa.container;
 
+import static org.jboss.as.jpa.messages.JpaLogger.JPA_LOGGER;
```

Comment:
Philippe, we usually reject changes to source files that are purely cosmetic.  Could you please remove TransactionScopedEntityManager.java from the pull request, so this can get merged.  Thanks!